### PR TITLE
Publish generated emails to kinesis

### PR DESCRIPTION
**Pull request by phamann**
- Create new emails kinesis stream via cloudformation*
- Create event source mapping to invoke `send-email` function via kinesis
- Add new `env` property to the data event data model so we can ensure emails are only sent in production.
- Add send email logic to `create-email` to publish generated email data to kinesis
- Refactor `send-email` index to process multiple records from kinesis
- Tests

**N.b**
This models the kinesis streams between get-users, create-email and send-email in cloudformation and the eventsource mappings to automate the invocations of said functions. It also renames the cloudformation script, hence all of the changes to that file. Sorry.

### CODE REVIEW

**Comment by phamann** [comment]
The are the important lines which have changed:
https://github.com/Financial-Times/next-myft-email/blob/send-email/templates/stack.json#L19-86

**Comment by keirog** [comment]
👍

**Comment by phamann** [comment]
This PR has completely changed since the last review, therefore can I please have another set of eyes in the morning :)

**Comment by laurieboyes** [comment]
Yep, I am on-board 👍

File: functions/send-email/index.js [hunk 1]
```diff
@@ -6,17 +6,36 @@ const sendToEmailService = require('./lib/send-to-email-service');
...
-	return sendToEmailService(config)
-		.catch(catchError)
-		.then(trackResponse);
+	// If single request process
+	if(!event.Records) {
+		return sendEmail(event);
+	}
+
+	// Otherwise a Kinesis request
+	// so process mutliple records
```
Comment (keirog) [inline functions/send-email/index.js]: ~~mutliple~~ multiple

**Comment by keirog** [comment]
👍

**Pull request merged**